ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/generator/BlockProposalTestUtil.java

```diff
@@ -255,6 +275,32 @@ private ExecutionPayload createExecutionPayload(
                                 List.of(
                                     dataStructureUtil.randomWithdrawal(),
                                     dataStructureUtil.randomWithdrawal()))))
+        // Eip4844 version
+        .or(
+            () ->
+                schema
+                    .toVersionEip4844()
+                    .map(
+                        schemaEip4844 ->
+                            schemaEip4844.create(
+                                parentHash,
+                                Bytes20.ZERO,
+                                dataStructureUtil.randomBytes32(),
+                                dataStructureUtil.randomBytes32(),
+                                dataStructureUtil.randomBytes256(),
+                                specVersion
+                                    .beaconStateAccessors()
+                                    .getRandaoMix(state, currentEpoch),
+                                newSlot,
+                                UInt64.valueOf(30_000_000L),
+                                UInt64.valueOf(30_000_000L),
+                                specVersion.miscHelpers().computeTimeAtSlot(state, newSlot),
+                                dataStructureUtil.randomBytes32(),
+                                UInt256.ONE,
+                                UInt64.ONE,
+                                dataStructureUtil.randomBytes32(),
+                                transactions.orElse(Collections.emptyList()),
+                                List.of())))
```

Comment:
Don't like some inconsistency here with Capella's withdrawals, but not sure, that we should do something with it on this stage

Reply:
I agree, I think we should have an empty list in both cases here, but I haven't tracked where the two withdrawals are actually required (if they even are).
I'd live this to upcoming tests